Pod/Classes/BONText.h

```diff
@@ -66,6 +66,9 @@ typedef NS_ENUM(NSUInteger, BONFigureSpacing) {
  */
 @property (nonatomic) CGFloat indentSpacer;
 
+@property (assign, nonatomic) NSUnderlineStyle underlineStyle;
```

Comment:
#req don't specify `assign`. This is technically a #pp, but one I feel very strongly about. Since the advent of ARC, `assign` is pretty much never a good choice for objects, and is the only option for primitives. Therefore, if I see `assign` in a project, I immediately assume that it should not be there.